sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -120,9 +126,196 @@ def swap_archive(config, fpath):
         os.makedirs(dname)
     return tc
 
+def preproc_or_load_mpi(obs_id, configs, dets, logger=None, 
+                        context=None, overwrite=False):
```

Comment:
I think you should downplay the "MPI" in this function name, and docstring.  Ok to mention it, but it's not really an essential to this function.  Lots of callers, parallel or not, might want to preproc_or_load for one of the standard logical groupings of dets.

So, just "preproc_or_load"?  Or "preproc_or_load_group"?

Reply:
Done.